run/deploy-configuration.js

```diff
@@ -14,6 +14,15 @@ module.exports = [
       'master',
     ),
   },
+  {
+    slashCommand: {
+      command: '/deploy-privatebin',
+      description: 'Déploie privatebin',
+      usage_hint: '/deploy-privatebin',
+    },
+    slackReturnText: 'Commande de déploiement de PrivateBin en production bien reçue.',
+    deployFunction: fromBranch('privatebin-deploy', ['pix-privatebin-production'], 'master'),
```

Comment:
```suggestion
    deployFunction: fromBranch('privatebin-deploy', ['pix-privatebin-production'], 'main'),
```
Le projet n'a pas de branch master : [privatebin-deploy#main](https://github.com/1024pix/privatebin-deploy)